cdap-ui/app/main.js

```diff
@@ -310,6 +310,8 @@ angular
       getVersion();
     }
 
+    $scope.copyrightYear = new Date().getFullYear();
```

Comment:
When we had to change this last year, it was said that we just need to include the latest year